.buildkite/scripts/run_job.py

```diff
@@ -122,7 +122,14 @@ def should_check_release(project_name):
     if should_check_release(args.project_name):
         latest_sha = get_sha1_for_tag("latest-sbt-release")
         commit_range = f"{latest_sha}..{local_head}"
-        check_release_file(commit_range)
+        changed_paths = get_changed_paths(commit_range, globs=change_globs)
+
+        if should_run_sbt_project(sbt_repo, args.project_name, changed_paths):
```

Comment:
I think I'm missing something... Why would the changed paths be different here than on line 112?

Reply:
Agreed - I think the problem is that `commit_range` (line 83) is using the `latest` tag rather than the new `latest-sbt-release` tag (?)

Reply:
And as we found yesterday, that tag is on the commit _before_ the version bump

Reply:
So the fix might be to compare to `latest-sbt-release` on line 83 instead?

Reply:
> Why would the changed paths be different here than on line 112?

The changed paths on line 112 are `latest..HEAD`. The changed paths here are `latest-sbt-release..HEAD`. Those two tags are set and managed differently – as Jamie says, one is before the version bump, the other is after.

(We created the `latest-sbt-release` tag yesterday to specifically point to the version bump commit.)

Reply:
I think that's right, which makes me wonder if we do need 2 separate tags - where would we be using `latest` if that's the comparison we were making?

Reply:
We use `latest` in weco-deploy, right?

Reply:
Is `latest` something we copied across from scala-libs but doesn't make sense here sans version bump?

Reply:
I _think_ that the `latest` in weco-deploy is an unrelated one to this git tag

Reply:
Note that even if we sort out the `latest` tag, I don't think we're out of the woods – there's about a 20–30 min delay between you merging to master and the internal_model being released/tagged, which could potentially break subsequent builds.

(I thought this was the issue with your PR yesterday.)

Reply:
I think you're right about that, yeah (although maybe it wasn't the case for the build yesterday, I waited until everything was done to merge that PR). Trying to think of a solution for that...

Reply:
I am tempted to comment this out until we can come up with a proper fix…

Reply:
ok thanks for the explanation...I did copy across from scala-libs and yeah, I don't think the `latest` tag was ever used before that in catalogue.
I think my understanding is still a bit fuzzy, but why do we want to hold off on this fix?I would still put this solution out if it works. it's still better even if we have a problem while the internal_model is being released irghth?

Reply:
I am possibly a little frustrated, having fought it all of yesterday. 😓 

I've pushed a change to compare to `latest-sbt-release`, are we happy for that to go in as a stopgap?

Reply:
sounds ok to me. Sorry about this